events/event/views.py

```diff
@@ -1,3 +1,79 @@
-from django.shortcuts import render
+import json
+import datetime
 
-# Create your views here.
+from django.utils.timezone import get_current_timezone
+from django.http.response import HttpResponseNotFound
+from django.http import JsonResponse, HttpResponse
+from django.views.generic.base import View
+from django.core import serializers
+
+from .models import Event, EventUserAssignment, User
+from .forms import EventCreateForm
+
+# Variabels for converting string to datetime
+tz = get_current_timezone()
+format = '%b %d %Y %I:%M%p'
+
+class EventView(View):
+    def get(self, request, pk=None):
+        if not pk:
```

Comment:
Get should return something ONLY when it gets identifier (object ID). It should never return everything from the database table.